service/version_bundles.go

```diff
@@ -0,0 +1,123 @@
+package service
+
+import (
+	"time"
+
+	"github.com/giantswarm/versionbundle"
+)
+
+func NewVersionBundles() []versionbundle.Bundle {
+	return []versionbundle.Bundle{
+		{
+			Changelogs: []versionbundle.Changelog{
+				{
+					Component:   "calico",
+					Description: "Calico version updated.",
+					Kind:        "changed",
+				},
+				{
+					Component:   "docker",
+					Description: "Docker version updated.",
+					Kind:        "changed",
+				},
+				{
+					Component:   "etcd",
+					Description: "Etcd version updated.",
+					Kind:        "changed",
+				},
+				{
+					Component:   "kubedns",
+					Description: "KubeDNS version updated.",
+					Kind:        "changed",
+				},
+				{
+					Component:   "kubernetes",
+					Description: "Kubernetes version updated.",
+					Kind:        "changed",
+				},
+				{
+					Component:   "nginx-ingress-controller",
+					Description: "Nginx-ingress-controller version updated.",
+					Kind:        "changed",
+				},
+			},
+			Components: []versionbundle.Component{
+				{
+					Name:    "calico",
+					Version: "2.6.2",
+				},
+				{
+					Name:    "docker",
+					Version: "1.12.6",
+				},
+				{
+					Name:    "etcd",
+					Version: "3.2.7",
+				},
+				{
+					Name:    "kubedns",
+					Version: "1.14.5",
+				},
+				{
+					Name:    "kubernetes",
+					Version: "1.8.1",
+				},
+				{
+					Name:    "nginx-ingress-controller",
+					Version: "0.9.0",
+				},
+			},
+			Dependencies: []versionbundle.Dependency{},
+			Deprecated:   false,
+			Name:         "aws-operator",
+			Time:         time.Date(2017, time.November, 29, 16, 16, 0, 0, time.UTC),
+			Version:      "0.1.0",
+			WIP:          true,
+		},
+		{
+			Changelogs: []versionbundle.Changelog{
+				{
+					Component:   "cloudformation",
+					Description: "First version of Cloud Formation resources.",
+					Kind:        "added",
+				},
+			},
+			Components: []versionbundle.Component{
+				{
+					Name:    "calico",
+					Version: "2.6.2",
+				},
+				{
+					Name:    "cloudformation",
+					Version: "0.1.0",
+				},
```

Comment:
Does that actually exist? What is `0.1.0` then? Maybe something we have to learn how to do it. I thought having a changelog is good enough.

Reply:
Still getting my head around the version bundle format.

It doesn't exist yet but we'll need to version the cloud config templates. So that could be the version but not certain yet.

Reply:
Yeah no worries. Same here. We should ask what the information is good for. We expose a `kubernetes` component and its actual upstream version to make users aware of the fact that we use a certain Kubernetes version. When we want to let the users know we use a specific component version then this is totally fine, but this should then be the version they know and are able to lookup upstream. In this case we would make up some version that is actually just some internal detail for us and thus would not provide any value to the customer. I am not sure if there is a cloud formation version or if Amazon just provides this concept as it is. I would maybe just drop the components here but keep the changelog since this can refer to anything we did and referring to Cloud Formation in the changelog is totally valid to me since this might be of interest for the user.

Reply:
Another reason to add the component here would be to make other operators be able to define a dependency on the component the aws-operator defines, but I think this does not apply here as well.